libraries/chain/include/eosio/chain/hs_proposal_info.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Should this file be located in hotstuff directory? If so, no need `hs_` in the name.

Reply:
I can move it. I thought that because it is a block header extension, it should be in `chain`, but I have no strong opinion on this.

Reply:
@heifner what do you think, should I move `hs_proposal_info.hpp` to the `hotstuff` subdirectory?

Reply:
Makes sense to move it to me.

Reply:
Done. 

By that rationale, `finalizer_set.hpp` should be in `hotstuff` as well. Should I move it in this PR?

Reply:
Please approve this PR, there are changes all over and I'll do it in a separate PR.